doc/source/structures/orbits/orbitable.rst

```diff
@@ -40,11 +40,6 @@ All objects that can move in orbit around other objects share some similar struc
      :attr:`ALTITUDE`        :ref:`scalar <scalar>` (m)
      :attr:`GEOPOSITION`     :struct:`GeoCoordinates`
      :attr:`PATCHES`         :struct:`List` of :struct:`Orbits <Orbit>`
-    ----------------------- --------------
-     The Following are deprecated (use apoapsis and periapsis on :attr:`OBT`)
-    --------------------------------------
-     :attr:`APOAPSIS`        :ref:`scalar <scalar>` (m)
-     :attr:`PERIAPSIS`       :ref:`scalar <scalar>` (m)
```

Comment:
Disagree because old scripts might be using it.

Reply:
Replaced!